castle/test/review_test.py

```diff
@@ -0,0 +1,18 @@
+import responses
+
+from castle.test import unittest
+from castle.review import Review
+
+
+class ReviewTestCase(unittest.TestCase):
+    @responses.activate
+    def test_retrieve(self):
+        response_text = 'review'
```

Comment:
can you use an actual sample payload here so we can be sure that the stringified Json is actually parsed